conceptnet5/support_data/jSIM/similarity_unambiguous/.~lock.score_adv_new_strict.csv#

```diff
@@ -0,0 +1 @@
+,mara,mara-Lenovo-YOGA-900S-12ISK,27.05.2018 00:24,file:///home/mara/.config/libreoffice/4;
```

Comment:
What are these `.~lock*` files?

Reply:
That's a very good question. I'll remove them.